tools/bin/ecflow/processes/main/meteorological/get_med_metfcst.py

```diff
@@ -0,0 +1,194 @@
+                 'pr': 'mm', 'pet': 'mm'}
+    old_units = {'vs': 'm s-1', 'sph': 'kg kg-1',
+                 'tmmx': 'K', 'tmmn': 'K', 'srad': 'W m-2',
+                 'pr': 'mm', 'pet': 'mm'}
+
+    # download metdata from http://thredds.northwestknowledge.net
+    for model in modelnames:
+        dlist = []
+        for var, name in varnames.items():
+            url = ('http://thredds.northwestknowledge.net:8080/thredds/dodsC/' +
+                   'NWCSC_INTEGRATED_SCENARIOS_ALL_CLIMATE/' +
+                   'cfsv2_metdata_90day/' +
+                   'cfsv2_metdata_forecast_%s_daily.nc' % (var))
+            print('Reading {0}'.format(url))
+            xds = xr.open_dataset(url)
+            if name == 'air_temperature':
+                # Change variable names so that tmmn and tmax are different
+                xds.rename({'air_temperature': var}, inplace=True)
+            dlist.append(xds)
+        merge_ds = xr.merge(dlist)
+        # MetSim requires time dimension be named "time"
+        merge_ds.rename({'day': 'time'}, inplace=True)
+        for var in ('tmmn', 'tmmx'):
+            units_in = cf_units.Unit(old_units[var])
+            units_out = cf_units.Unit(new_units[var])
+            # Perform units conversion
+            units_in.convert(merge_ds[var].values[:], units_out, inplace=True)
+            # Fix _FillValue after unit conversion
+            merge_ds[var].values[merge_ds[var].values < -30000] = -32767.
+        # MetSim requires time dimension be named "time"
+        merge_ds = merge_ds.transpose('time', 'lat', 'lon')
+        merge_ds = recreate_attrs(merge_ds)
+        # Make sure tmax >= tmin always
+        tmin = np.copy(merge_ds['tmmn'].values)
+        tmax = np.copy(merge_ds['tmmx'].values)
+        swap_values = ((tmin > tmax) & (tmax != -32767.))
+        merge_ds['tmmn'].values[swap_values] = tmax[swap_values]
+        merge_ds['tmmx'].values[swap_values] = tmin[swap_values]
+        time = merge_ds['time']
+        end_date = netCDF4.num2date(time[-1], time.units, time.calendar)
+        start_date = datetime.strptime(config_dict['MONITOR']['End_Date'],
+                                       '%Y-%m-%d') + timedelta(days=1)
+        start_date_format = start_date.strftime('%Y-%m-%d')
+        end_date_format = end_date.strftime('%Y-%m-%d')
+        vic_save_state_format = (end_date + timedelta(days=1)).strftime('%Y-%m-%d')
+        # replace start date, end date and met location in the configuration
+        # file
+        kwargs = {'START_DATE': config_dict['MONITOR']['Start_Date'],
+                  'END_DATE': config_dict['MONITOR']['End_Date'],
+                  'VIC_SAVE_STATE': config_dict['MONITOR']['vic_save_state'],
+                  'MED_START_DATE': start_date_format,
+                  'MED_END_DATE': end_date_format,
+                  'MED_VIC_SAVE_STATE': vic_save_state_format,
+                  'SEAS_START_DATE': -9999,
```

Comment:
Why `-9999` for these three?

Reply:
Addressed with comment: "SEAS_* variables will be defined in get_seasonal_metfcst.py. Replace those keywords with dummy variable (-9999) so that replace_var_pythonic_config() won't fail."